CMake/kwiver-config-build.cmake.in

```diff
@@ -14,9 +14,8 @@ set(CMAKE_MODULE_PATH "@KWIVER_CMAKE_DIR@" "${CMAKE_MODULE_PATH}")
 
 # KWIVER include directory
 set(KWIVER_INCLUDE_DIRS
-  "@KWIVER_SOURCE_DIR@" "@KWIVER_SOURCE_DIR@/vital"
-  "@KWIVER_BINARY_DIR@" "@KWIVER_BINARY_DIR@/vital"
-  "@KWIVER_BINARY_DIR@/vital/kwiversys"
```

Comment:
Before removing this, it would be good to test if KWIVER itself [can build without it](https://github.com/Kitware/kwiver/blob/b53f7debf2ffbbf3c31957ad1159f0793e488652/CMakeLists.txt#L79). That should at least give a hint if there are public headers that depend on this.

Reply:
Ping?

BTW, there *are* public headers including stuff from kwiversys, but hopefully they all include "kwiversys/whatever.h". (At least the one I ran into seemed okay in that respect.)

Reply:
All kwiversys headers in kwiver and vidtk are included as <kwiversys/whatever.h> with the exception of the files internal to kwiversys. Everything does build though considering this error has been less consistently reproducible that I would have thought, I'm happy to err on the side of caution here and only remove the include_directories that are known to cause the issue with signal.h, e.g. vital.